draft-ietf-masque-h3-datagram.md

```diff
@@ -623,68 +314,30 @@ recent draft version from the intersection set. This ensures that both
 endpoints negotiate the same draft version.
 
 
-# The Sec-Use-Datagram-Contexts HTTP Header {#context-hdr}
+# The Sec-Capsule-Protocol Header
 
-Endpoints indicate their support for datagram contexts by sending the
-Sec-Use-Datagram-Contexts header with a value of ?1. If the header is missing
-or has a value different from ?1, that indicates that its sender does not wish
-to use datagram contexts. Endpoints that wish to use datagram contexts SHALL
-send the Sec-Use-Datagram-Contexts header with a value of ?1 on requests and
-responses that use the capsule protocol.
+Definitions of new HTTP Methods or of new HTTP Upgrade Tokens that use the
+Capsule Protocol MAY use the Sec-Capsule-Protocol header field to simplify
+intermediary processing.
 
-"Sec-Use-Datagram-Contexts" is an Item Structured Header {{!RFC8941}}. Its
-value MUST be a Boolean, its ABNF is:
+Endpoints indicate that the Capsule Protocol is in use on the data stream by
+sending the Sec-Capsule-Protocol header field with a value of ?1. Intermediaries
+MAY use this header field to allow processing of HTTP Datagrams for unknown HTTP
+methods or unknown HTTP Upgrade Tokens.
+
+"Sec-Capsule-Protocol" is an Item Structured Header {{!RFC8941}}. Its value MUST
+be a Boolean, its ABNF is:
 
 ~~~
-Sec-Use-Datagram-Contexts = sf-boolean
+Sec-Capsule-Protocol = sf-boolean
```

Comment:
What are the semantics of this boolean? I don't see them defined.

Reply:
Good point, I've added text to clarify the semantics.